feature/collectionSheet/src/commonMain/kotlin/com/mifos/feature/individualCollectionSheet/individualCollectionSheetDetails/IndividualCollectionSheetDetailsScreen.kt

```diff
@@ -246,18 +248,16 @@ private fun IndividualCollectionSheetItem(
                     Text(
                         text = it,
                         style = MaterialTheme.typography.bodyLarge,
-
                     )
                 }
                 Row {
                     Text(
                         text = stringResource(Res.string.feature_collection_sheet_total_due),
                         style = MaterialTheme.typography.bodyMedium,
-
                     )
                     Spacer(modifier = Modifier.width(16.dp))
                     Text(
-                        text = client.loans?.get(index)?.totalDue.toString(),
+                        text = (loan?.totalDue ?: 0).toString(),
```

Comment:
- the zero should be in this format `0.0`
- i see this screen there are things that need to be taken care of at viewmodel and the screen should only display to user, but no worries for now we should come back for all of these